src/SprykerSdk/Acp/Translation/Builder/AppTranslationBuilder.php

```diff
@@ -21,10 +24,27 @@ public function createTranslation(AppTranslationRequestTransfer $appTranslationR
     {
         $appTranslationResponseTransfer = new AppTranslationResponseTransfer();
 
-        $this->writeToFile(
-            $appTranslationRequestTransfer->getTranslationFileOrFail(),
-            $appTranslationRequestTransfer->getTranslations(),
-        );
+        try {
+            $this->writeToFile(
+                $appTranslationRequestTransfer->getTranslationFileOrFail(),
+                $appTranslationRequestTransfer->getTranslations(),
+            );
+        } catch (Exception | Error $error) {
```

Comment:
If it's really necessary, could you use `\Throwable` instead, like 
```suggestion
        } catch (Throwable $error) {
```

Reply:
Fixed